cmd/lncli/main.go

```diff
@@ -65,24 +68,41 @@ func getClient(ctx *cli.Context) (lnrpc.LightningClient, func()) {
 
 func getClientConn(ctx *cli.Context, skipMacaroons bool) *grpc.ClientConn {
 	lndDir := cleanAndExpandPath(ctx.GlobalString("lnddir"))
+	chain := ctx.GlobalString("chain")
+	network := ctx.GlobalString("network")
```

Comment:
We should probably explicitly provide flags for each chain and network like we do in `lnd`. Could have something like `lncli --litecoin --testnet`, `lncli --bitcoin --simnet`, etc.

Reply:
The defaults should stay as they are currently, bitcoin mainnet.